[CKLS-21158] Replace the symfony/symfony package

Alex-Caepio opened the pull request.

Alex-Caepio committed:
[CKLS-21158] Replaced the symfony/symfony package with the symfony/flex

Alex-Caepio committed:
[CKLS-21158] Replaced the symfony/symfony package with the symfony/flex

Alex-Caepio committed:
[CKLS-21158] Replaced the symfony/symfony package with the symfony/flex

Alex-Caepio committed:
[CKLS-21158] Replaced the symfony/symfony package with the symfony/flex

irena-zaz commented on composer.json:
```diff
@@ -21,11 +21,28 @@
   "require": {
     "php": ">=7.2",
```
same for Symfony 5.4 we need php >=7.2.5
```suggestion
    "php": ">=7.2.5",
```

Alex-Caepio replied:
Upgraded the PHP version to 7.2.5

irena-zaz commented:
We do not run the unit tests, in the pipelines, will you confirm that they all pass?

Alex-Caepio committed:
[CKLS-21158] Update dependencies and PHP version constraint.

Increased the minimum PHP version to 7.2.5 in composer.json. Updated several dependencies in composer.lock, including `matomo/device-detector`, `symfony/flex`, and `myclabs/deep-copy`, among others, to their latest compatible versions.
